src/Microsoft.IdentityModel.Tokens/AsymmetricAdapter.cs

```diff
@@ -395,244 +340,6 @@ private bool VerifyWithRsaCryptoServiceProviderProxyWithLength(byte[] bytes, int
     #endif
 
 #endif
-#endregion
```

Comment:
which start of region does it relate to? (I probably missed it)
(we can't remove the #endregion without removing the #region )?

Reply:
The diff shows a different endregion being removed; it effectively doesn't matter which one is removed.